config/config.go

```diff
@@ -217,7 +216,7 @@ type ScrapeConfig struct {
 	// Indicator whether the scraped metrics should remain unmodified.
 	HonorLabels bool `yaml:"honor_labels,omitempty"`
 	// A set of query parameters with which the target is scraped.
-	Params url.Values `yaml:"params,omitempty"`
+	Params map[string]string `yaml:"params,omitempty"`
```

Comment:
Ok, I'd not propagate that constraint all the way until here then (for the aforementioned reasons). Options:

1. Only expose the last of each param occurrence during relabeling. If you want to relabel repeated params, you're just a bit screwed, but it's probably fine for most use cases.
2. Concatenate them with the relabel concatenation separator (hard to implement because you'd already have to use something different than a map to pass labels into around before relabeling - so, nah).
3. Make them unique in the label map by appending numeric suffixes in case of repeated arguments. Not sure how usable/useful this would be in practice during relabeling.

Probably just 1?